aequilibrae/project/data/matrices.py

```diff
@@ -1,22 +1,22 @@
 import os
 from os.path import isfile, join
-from sqlite3 import Connection
 import pandas as pd
 from aequilibrae.project.table_loader import TableLoader
 from aequilibrae.matrix import AequilibraeMatrix
 from aequilibrae.project.data.matrix_record import MatrixRecord
+from aequilibrae.starts_logging import logger
+from aequilibrae.project.database_connection import database_connection, environ_var
 
 
 class Matrices:
     """Gateway into the matrices available/recorded in the model"""
     __items = {}
     __fields = []
 
-    def __init__(self, project):
-        self.__project = project
-        self.conn = project.conn  # type: Connection
+    def __init__(self):
+        self.conn = database_connection()
         self.curr = self.conn.cursor()
-        self.fldr = os.path.join(project.project_base_path, 'matrices')
+        self.fldr = os.path.join(os.environ.get(environ_var), 'matrices')
```

Comment:
where does environ_var come from?

Reply:
oh it's imported? - is it a CONSTANT?

Reply:
Yes, it is

Reply:
doesn't pep8 say that it should be ENVIRON_VAR?

Reply:
I don't think so (just searched and could not find anything on it)

Reply:
why oh why do you doubt me?
![image](https://user-images.githubusercontent.com/151124/100949450-0a3a8900-3556-11eb-919a-7321f5529516.png)

Reply:
It has to do with being a constant!!! I was looking for references to environment variables!!!

And I doubt you just to keep you sharp! Fixed